script/build.js

```diff
@@ -395,7 +395,6 @@ if (options.buildtype !== 'development') {
 
 smith.use(redirect({
   '/2015/11/11/why-we-are-designing-in-beta.html': '/2015/11/11/why-we-are-designing-in-beta/',
-  '/disability-benefits/apply/': '/disability-benefits/apply/',
```

Comment:
Is there a reason to remove this instead of correcting it to `'/disability-benefits/apply-for-disability-benefits/': '/disability-benefits/apply/'`? (And if so, would the same be true for the following line?)